fontforge/sfd.c

```diff
@@ -1841,11 +1842,57 @@ return;
     putc('\n',sfd);
 }
 
+static void SFDDumpOtfSubFeatNames(FILE *sfd, SplineFont *sf, uint32_t tag, enum otffn_field field, uint32_t lang) {
+    struct otffeatname *fn;
+    struct otfname *on;
+    bool cvfound = false;
+
+	for ( fn=sf->feat_names; fn!=NULL; fn=fn->next ) {
+		if ( fn->tag == tag && fn->field == field ) {
+			for ( on=fn->names; on!=NULL; on=on->next ) {
+				if ( on->lang == lang ) {
+					SFDDumpUTF7Str(sfd, on->name);
+					cvfound = true;
+				}
+			}
+		}
+	}
+	if ( !cvfound ) {
+		fprintf( sfd, "\"\"" );
+	}
+}
+static int SFDDumpOtfCountFeatNames(FILE *sfd, SplineFont *sf, uint32_t tag, uint32_t lang) {
+    struct otffeatname *fn;
+    struct otfname *on;
+    bool cvfound;
+	int i;
+
+	for ( i = 0; i < 65536-256; ++i ) {
+		cvfound = false;
+		for ( fn=sf->feat_names; fn!=NULL; fn=fn->next ) {
+			if ( fn->tag == tag && fn->field == otffn_paramname_begin + i ) {
+				for ( on=fn->names; on!=NULL; on=on->next ) {
+					if ( on->lang == lang ) {
+						cvfound = true;
+					}
+				}
+			}
+		}
+		if ( !cvfound ) {
+			break;
+		}
+	}
+	return i;
+}
 static void SFDDumpOtfFeatNames(FILE *sfd, SplineFont *sf) {
     struct otffeatname *fn;
     struct otfname *on;
+	bool cv[100] = {false};
+	int cvnum, cvcount, i;
+	char prefix[3] = {0};
```

Comment:
@MihailJP `prefix` is added but unused and produces a warning. Should I submit a PR removing the line or does it point to another problem with the patch?